$NetBSD: patch-src_qdoc_configure.json,v 1.1 2023/12/19 13:01:37 thor Exp $

Add config switch to disable llvm dependency. Seems to be that simple.

--- src/qdoc/configure.json.orig	2021-07-12 16:32:50.358821257 +0000
+++ src/qdoc/configure.json
@@ -1,5 +1,10 @@
 {
     "module": "qdoc",
+    "commandline": {
+        "options": {
+            "qdoc": "boolean"
+        }
+    },
     "tests": {
         "libclang": {
             "label": "libclang",
